utils/land-patterns.stanza

```diff
@@ -911,22 +965,22 @@ public pcb-landpattern INDC1005X55 :
 
 public pcb-landpattern CAPT1005X55 :
   make-ipc-two-pin-landpattern("0402")
-  layer(Silkscreen("pol", Top)) = PlusSymbol(-1.7, 0.0)
+  draw-polarity-marker(IndexRef(Ref("p"), 1))
```

Comment:
After the change to `draw-polarity-marker` to take `JITXObject`, you can call it like this:

```
draw-polarity-marker(self.p[1])
```

Reply:
It seems that the `two-pin-landpattern` should directly generate the polarity marker itself instead of relying upon callers to do it. Especially since it already takes a `polarized?` argument.

Reply:
This code does not call the two pin landpattern generator (and doing so would break it, because callers of this generator don't use `a` and `c` as the pad names).

Reply:
Added the changes